### PR TITLE
Allow UTF-8 elements in file, directories, arguments

### PR DESCRIPTION
**Pull request by holzschu**
With help from @IMcD23 this small modification allows the user to use UTF-8 characters in commands  and arguments.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+Terminal.xcodeproj/project.pbxproj
```
Comment (louisdh) [inline .gitignore]: This should not needed to be added to gitignore. You can delete this (old) file locally.

File: Dependencies/ios_system/ios_system.m [hunk 2]
```diff
@@ -302,6 +302,10 @@ static void initializeCommandList()
       @"grep"   : [NSArray arrayWithObjects:@"libtext.dylib", @"grep_main", @"0123456789A:B:C:D:EFGHIJMLOPSRUVZabcd:e:f:hilm:nopqrsuvwxXy", @"file", nil],
       @"egrep"  : [NSArray arrayWithObjects:@"libtext.dylib", @"grep_main", @"0123456789A:B:C:D:EFGHIJMLOPSRUVZabcd:e:f:hilm:nopqrsuvwxXy", @"file", nil],
       @"fgrep"  : [NSArray arrayWithObjects:@"libtext.dylib", @"grep_main", @"0123456789A:B:C:D:EFGHIJMLOPSRUVZabcd:e:f:hilm:nopqrsuvwxXy", @"file", nil],
+      // network commands:
+      @"ping"     : [NSArray arrayWithObjects:@"libnetwork_ios.dylib", @"ping_main", @"AaB:b:Cc:DdfG:g:h:I:i:k:K:Ll:M:m:noP:p:QqRrS:s:T:t:vW:z:", @"no", nil],
+      @"nc"       : [NSArray arrayWithObjects:@"libnetwork_ios.dylib", @"netcat_main", @"46AacDCb:dEhi:jFG:H:I:J:K:L:klMnN:Oop:rSs:T:tUuvw:X:x:z", @"no", nil],
+      @"nslookup" : [NSArray arrayWithObjects:@"libnetwork_ios.dylib", @"nslookup_main", @"", @"no", nil],
```
Comment (louisdh) [inline Dependencies/ios_system/ios_system.m]: Do these network commands belong to this PR?

Comment (holzschu) [reply Dependencies/ios_system/ios_system.m]: No, they don't. Do you want me to remove them?

Comment (louisdh) [reply Dependencies/ios_system/ios_system.m]: Yes please, trying to keep things ordered 🙂

**Comment by louisdh** [comment]
Thank you for your quick responses!

**Pull request merged**